src/components/new Movie/index.tsx

```diff
@@ -0,0 +1,84 @@
+import React, { useState } from "react"
+import FormControl from "react-bootstrap/FormControl"
+import InputGroup from "react-bootstrap/InputGroup"
+import Button from "react-bootstrap/esm/Button"
+
+/*{ "Title": "Batman Begins"
+, "Year": "2005"
+, "imdbID": "tt0372784"
+, "Type": "movie"
+, "Poster": "https://m.media-amazon.com/images/M/MV5BZmUwNGU2ZmItMmRiNC00MjhlLTg5YWUtODMyNzkxODYzMmZlXkEyXkFqcGdeQXVyNTIzOTk5ODM@._V1_SX300.jpg",
+ "rank": 4 }*/
+
+ interface IProp{
+     addMovie:Function
+     visible:string
+ }
+
+export default function NewMovie(props:IProp){
+
+    const EmptyString = ""
+
+    const [movie,setMovie]=useState({Title:EmptyString,Year:EmptyString,imdbID:EmptyString,Type:EmptyString,Poster:EmptyString,rank:0})
+
+    const {addMovie,visible} =props
+
+    function getMovie(movie:{Title:string,Year:string,imdbID:string,Type:string,Poster:string,rank:number}){
+        
+        if(movie.Title === EmptyString|| movie.Year === EmptyString ||movie.imdbID === EmptyString || movie.Type === EmptyString ||movie.Poster ===EmptyString)
+            return false
+        addMovie(movie)
+
+    }
+    
+    return <div style={{border:"solid black 2px",padding:"10px",borderRadius:"20px" ,display:visible}}>
+
+        <h1>Add Movie</h1>
+        
+        <InputGroup className="mb-1">
+        <FormControl
+        placeholder="Enter Title"
+        onChange={(e)=>{setMovie({...movie,Title:e.target.value})}}>
+        </FormControl>
+        </InputGroup>
+        
+        <InputGroup className="mb-1">
+        <input 
+         type="number"
+         className="form-control"
+         placeholder="Enter Year"
+         onChange={(e)=>{setMovie({...movie ,Year:e.target.value})}}
```

Comment:
you can use the name of the input as a key and the value of the input as the value
and then assign it 
form[key] = value